geode-core/src/integrationTest/java/org/apache/geode/distributed/LocatorLauncherLocalIntegrationTest.java

```diff
@@ -74,6 +77,21 @@ public void startReturnsOnline() {
     assertThat(launcher.start().getStatus()).isEqualTo(ONLINE);
   }
 
+  @Test
+  public void whenLocatorIsPresetThenItWillNotReadThePropertyFiles() {
+    launcher = givenLocatorLauncher();
+
+    assertThat(launcher.start().getStatus()).isEqualTo(ONLINE);
+    LocatorLauncher locatorLauncher = spy(launcher);
+    locatorLauncher.status();
+    /*
+     * The verification done here is that if the locator is present
+     * then we are not re-constructing the DistributedConfigImp which is
+     * triggered when we call the getProperties()
```

Comment:
Sorry to nit-pick but this comment is misleading. Calling `getProperteis()` doesn't trigger construction of `DistributedConfigImpl`. Suggest something like:

> We want to verify that no `DistributedConfigImpl` is constructed. But that's hard/impossible to mock.
> We happen to know that in the case where `SSLConfigurationFactory.getSSLConfigForComponent()`
> constructs one,  `getProperties()` is called. That's why we're asserting `getProperties()` isn't called.

Reply:
the new comment is accurate thanks!